Gemfile

```diff
@@ -0,0 +1,2 @@
+source "https://rubygems.org"

```

Comment:
Just of curiosity, were these just copied over from the front-end apps or is there a real need for them? (talking about the `Gemfile*` files.